micrometer-tracing-tests/micrometer-tracing-integration-test/src/test/java/io/micrometer/tracing/contextpropagation/TestBaggageThreadLocalAccessor.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 VMware, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micrometer.tracing.contextpropagation;
+
+import java.util.Map;
+
+public class TestBaggageThreadLocalAccessor {
```

Comment:
Can you add javadoc with a short description and a `@since` tag?

Reply:
But that's a test class